pkg/values/list.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"reflect"
 
+
```

Comment:
Could you remove this additional line, both here and in map.go @HenryNguyen5 ? Otherwise LGTM